front/components/vaults/VaultResourcesList.tsx

```diff
@@ -62,11 +62,12 @@ const getTableColumns = ({
 }: {
   isManaged: boolean;
   isSystemVault: boolean;
-}) => {
-  const nameColumn = {
+}): ColumnDef<RowData, any>[] => {
+  const nameColumn: ColumnDef<RowData, string> = {
```

Comment:
do you need to add these typings ? they should be infered ?

Reply:
ok just saw the error, but you don't need the `ColumnDef<RowData, any>[]`

Reply:
Good point, removing it 👌🏼